x/superfluid/keeper/msg_server.go

```diff
@@ -97,3 +97,27 @@ func (server msgServer) LockAndSuperfluidDelegate(goCtx context.Context, msg *ty
 		ID: lockupRes.ID,
 	}, err
 }
+
+func (server msgServer) UnPoolWhitelistedPool(goCtx context.Context, msg *types.MsgUnPoolWhitelistedPool) (*types.MsgUnPoolWhitelistedPoolResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
```

Comment:
Lets add a height gating here. `if ctx.BlockHeight() < vx.UpgradeHeight  {return nil, errors.New("message not activated"))}`

Will need to confirm thats state compatible until upgrade height

Reply:
yeah good catch! We can't use vx package directly due to import cycle, going to put this as antoher constant in keeper for now